src/plugins/ui_feature/find_and_replace.ts

```diff
@@ -237,9 +244,13 @@ export class FindAndReplacePlugin extends UIPlugin {
     }
 
     const sheetId = this.getters.getActiveSheetId();
-    const cell = this.getters.getCell({ sheetId, ...selectedMatch });
     const { col, row } = selectedMatch;
 
+    if (this.getters.getArrayFormulaSpreadingOn({ sheetId, col, row })) {
+      return;
+    }
+
+    const cell = this.getters.getCell({ sheetId, ...selectedMatch });
```

Comment:
Alternatively to the use `getArrayFormulaSpreadingOn` we could check that cell has a content. I think it could make sense to compare the performances?